pkg/sink/kafka/v2/admin.go

```diff
@@ -233,7 +221,7 @@ func (a *admin) CreateTopic(
 	}
 
 	for _, err := range response.Errors {
-		if err != nil {
+		if err != nil && !errors.Is(err, kafka.TopicAlreadyExists) {
```

Comment:
We should have a manual test to check if `errors.Is()` is effective. In earlier release, we made a wrong assumption and we finally adopt the string matching method. 
Ref: https://github.com/pingcap/tiflow/blob/release-6.5/cdc/sink/mq/manager/kafka_manager.go#L283

Reply:
/cc @hi-rustin